<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>978</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>CGAL Polyhedron demo</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="Polyhedron_3.qrc" >
    <normaloff>:/cgal/icons/resources/cgal_logo.xpm</normaloff>:/cgal/icons/resources/cgal_logo.xpm</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" name="gridLayout" >
    <item row="0" column="1" >
     <widget class="Viewer" native="1" name="viewer" />
    </item>
    <item row="0" column="0" >
     <widget class="QDockWidget" name="polyhedraDockWidget" >
      <property name="locale" >
       <locale country="UnitedStates" language="English" />
      </property>
      <property name="windowTitle" >
       <string>Polyhedra</string>
      </property>
      <widget class="QWidget" name="dockWidgetContent" >
       <layout class="QGridLayout" name="gridLayout_2" >
        <item row="0" column="0" >
         <layout class="QVBoxLayout" name="verticalLayout" >
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" >
            <item>
             <widget class="QToolButton" name="addButton" >
              <property name="text" >
               <string>+</string>
              </property>
              <property name="icon" >
               <iconset resource="Polyhedron_3.qrc" >
                <normaloff>:/cgal/icons/plus</normaloff>:/cgal/icons/plus</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="removeButton" >
              <property name="text" >
               <string>-</string>
              </property>
              <property name="icon" >
               <iconset resource="Polyhedron_3.qrc" >
                <normaloff>:/cgal/icons/minus</normaloff>:/cgal/icons/minus</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="duplicateButton" >
              <property name="text" >
               <string>...</string>
              </property>
              <property name="icon" >
               <iconset resource="Polyhedron_3.qrc" >
                <normaloff>:/cgal/icons/duplicate</normaloff>:/cgal/icons/duplicate</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTreeView" name="treeView" >
            <property name="editTriggers" >
             <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
            </property>
            <property name="alternatingRowColors" >
             <bool>true</bool>
            </property>
            <property name="indentation" >
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>978</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="actionLoadPolyhedron" />
    <addaction name="actionErasePolyhedron" />
    <addaction name="actionEraseAll" />
    <addaction name="actionDuplicatePolyhedron" />
    <addaction name="actionSaveAs" />
    <addaction name="separator" />
    <addaction name="actionQuit" />
   </widget>
   <widget class="QMenu" name="menuEdit" >
    <property name="title" >
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionActivatePolyhedron" />
    <addaction name="actionSetPolyhedronA" />
    <addaction name="actionSetPolyhedronB" />
   </widget>
   <widget class="QMenu" name="menuOperations" >
    <property name="title" >
     <string>&amp;Operations</string>
    </property>
    <widget class="QMenu" name="menuSubdivision" >
     <property name="title" >
      <string>&amp;Subdivision</string>
     </property>
     <addaction name="actionLoop" />
     <addaction name="actionCatmullClark" />
     <addaction name="actionSqrt3" />
    </widget>
    <widget class="QMenu" name="menu_Boolean_operations" >
     <property name="title" >
      <string>&amp;Boolean operations</string>
     </property>
     <addaction name="actionUnion" />
     <addaction name="actionIntersection" />
     <addaction name="actionDifference" />
    </widget>
    <widget class="QMenu" name="menuParameterization" >
     <property name="title" >
      <string>Parameterization</string>
     </property>
     <addaction name="actionMVC" />
     <addaction name="actionDCP" />
    </widget>
    <widget class="QMenu" name="menuPCA" >
     <property name="title" >
      <string>PCA</string>
     </property>
     <addaction name="actionFitLine" />
     <addaction name="actionFitPlane" />
    </widget>
    <addaction name="menuPCA" />
    <addaction name="actionEstimateCurvature" />
    <addaction name="actionSelfIntersection" />
    <addaction name="actionConvexHull" />
    <addaction name="actionKernel" />
    <addaction name="menuSubdivision" />
    <addaction name="actionSimplify" />
    <addaction name="actionRemeshing" />
    <addaction name="menu_Boolean_operations" />
    <addaction name="menuParameterization" />
    <addaction name="actionInsideOut" />
   </widget>
   <widget class="QMenu" name="menuView" >
    <property name="title" >
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menuDockWindows" >
     <property name="title" >
      <string>Dock windows</string>
     </property>
     <addaction name="dummyAction" />
    </widget>
    <addaction name="actionAntiAliasing" />
    <addaction name="actionViewEdges" />
    <addaction name="menuDockWindows" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuEdit" />
   <addaction name="menuOperations" />
   <addaction name="menuView" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="actionQuit" >
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSimplify" >
   <property name="text" >
    <string>&amp;Simplification</string>
   </property>
  </action>
  <action name="actionCatmullClark" >
   <property name="text" >
    <string>&amp;Catmull-Clark</string>
   </property>
  </action>
  <action name="actionKernel" >
   <property name="text" >
    <string>&amp;Kernel</string>
   </property>
  </action>
  <action name="actionUnion" >
   <property name="text" >
    <string>&amp;Union</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+O, U</string>
   </property>
  </action>
  <action name="actionIntersection" >
   <property name="text" >
    <string>&amp;Intersection</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+O, I</string>
   </property>
  </action>
  <action name="actionDifference" >
   <property name="text" >
    <string>&amp;Difference</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+O, D</string>
   </property>
  </action>
  <action name="actionFitPlane" >
   <property name="text" >
    <string>Fit &amp;plane</string>
   </property>
  </action>
  <action name="actionFitLine" >
   <property name="text" >
    <string>Fit &amp;line</string>
   </property>
  </action>
  <action name="actionEstimateCurvature" >
   <property name="text" >
    <string>&amp;Curvature estimation</string>
   </property>
  </action>
  <action name="actionLoadPolyhedron" >
   <property name="icon" >
    <iconset resource="Polyhedron_3.qrc" >
     <normaloff>:/cgal/icons/plus</normaloff>:/cgal/icons/plus</iconset>
   </property>
   <property name="text" >
    <string>&amp;Load...</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionErasePolyhedron" >
   <property name="icon" >
    <iconset resource="Polyhedron_3.qrc" >
     <normaloff>:/cgal/icons/minus</normaloff>:/cgal/icons/minus</iconset>
   </property>
   <property name="text" >
    <string>&amp;Erase</string>
   </property>
   <property name="shortcut" >
    <string>Del</string>
   </property>
  </action>
  <action name="actionDuplicatePolyhedron" >
   <property name="icon" >
    <iconset resource="Polyhedron_3.qrc" >
     <normaloff>:/cgal/icons/duplicate</normaloff>:/cgal/icons/duplicate</iconset>
   </property>
   <property name="text" >
    <string>&amp;Duplicate</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionSqrt3" >
   <property name="text" >
    <string>&amp;Sqrt3</string>
   </property>
  </action>
  <action name="actionAntiAliasing" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>&amp;Antialiasing</string>
   </property>
  </action>
  <action name="dummyAction" >
   <property name="text" >
    <string>n/a</string>
   </property>
  </action>
  <action name="actionConvexHull" >
   <property name="text" >
    <string>&amp;Convex hull</string>
   </property>
  </action>
  <action name="actionEraseAll" >
   <property name="text" >
    <string>&amp;Erase all</string>
   </property>
  </action>
  <action name="actionOptions" >
   <property name="text" >
    <string>&amp;Options...</string>
   </property>
  </action>
  <action name="actionLoop" >
   <property name="text" >
    <string>&amp;Loop</string>
   </property>
  </action>
  <action name="actionSaveAs" >
   <property name="text" >
    <string>Save &amp;as...</string>
   </property>
  </action>
  <action name="actionSave" >
   <property name="text" >
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionSaveAll" >
   <property name="text" >
    <string>Save a&amp;ll</string>
   </property>
  </action>
  <action name="actionMergeAll" >
   <property name="text" >
    <string>Mer&amp;ge all</string>
   </property>
  </action>
  <action name="actionMerge" >
   <property name="text" >
    <string>&amp;Merge</string>
   </property>
  </action>
  <action name="actionSelfIntersection" >
   <property name="text" >
    <string>Self-&amp;intersection</string>
   </property>
  </action>
  <action name="actionSelectAll" >
   <property name="text" >
    <string>Select &amp;all</string>
   </property>
  </action>
  <action name="actionSelectNone" >
   <property name="text" >
    <string>Select &amp;none</string>
   </property>
  </action>
  <action name="actionSelectInvert" >
   <property name="text" >
    <string>&amp;Invert selection</string>
   </property>
  </action>
  <action name="actionViewEdges" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Superimpose edges</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionActivatePolyhedron" >
   <property name="text" >
    <string>Activate selected polyhedron</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Space</string>
   </property>
  </action>
  <action name="actionSetPolyhedronA" >
   <property name="text" >
    <string>Set polyhedron A</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionSetPolyhedronB" >
   <property name="text" >
    <string>Set polyhedron B</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionInsideOut" >
   <property name="text" >
    <string>Inside-out</string>
   </property>
  </action>
  <action name="actionRemeshing" >
   <property name="text" >
    <string>Remeshing</string>
   </property>
  </action>
  <action name="actionMVC" >
   <property name="text" >
    <string>Mean value coordinates</string>
   </property>
  </action>
  <action name="actionDCP" >
   <property name="text" >
    <string>Discrete conformal maps</string>
   </property>
  </action>
  <action name="actionExplode" >
   <property name="text" >
    <string>Explode</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Viewer</class>
   <extends>QWidget</extends>
   <header>Viewer.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Polyhedron_3.qrc" />
 </resources>
 <connections/>
</ui>
